src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -1656,6 +1656,124 @@ public function get_test_https_status() {
+
+		$result = array(
+			'badge'       => array(
+				'label' => __( 'Performance' ),
+				'color' => 'green',
+			),
+			'description' => wp_kses_post( $description ),
+			'test'        => 'page_cache',
+			'status'      => 'good',
+			'label'       => '',
+			'actions'     => sprintf(
+				'<p><a href="%1$s" target="_blank" rel="noopener noreferrer">%2$s <span class="screen-reader-text">%3$s</span><span aria-hidden="true" class="dashicons dashicons-external"></span></a></p>',
+				__( 'https://wordpress.org/support/article/optimization/#Caching' ),
+				__( 'Learn more about page caching' ),
+				/* translators: Accessibility text. */
+				__( '(opens in a new tab)' )
+			),
+		);
+
+		$page_cache_detail = $this->get_page_cache_detail();
+
+		if ( is_wp_error( $page_cache_detail ) ) {
+			$result['badge']['color'] = 'orange';
+			$result['label']          = __( 'Unable to detect the presence of page caching' );
+			$result['status']         = 'recommended';
+			$error_info               = sprintf(
+			/* translators: 1 is error message, 2 is error code */
+				__( 'Unable to detect page caching due to possible loopback request problem. Please verify that the loopback request test is passing. Error: %1$s (Code: %2$s)' ),
+				$page_cache_detail->get_error_message(),
+				$page_cache_detail->get_error_code()
+			);
+			$result['description'] = wp_kses_post( "<p>$error_info</p>" ) . $result['description'];
+			return $result;
+		}
+
+		$result['status'] = $page_cache_detail['status'];
+
+		switch ( $page_cache_detail['status'] ) {
+			case 'recommended':
+				$result['badge']['color'] = 'orange';
+				$result['label']          = __( 'Page caching is not detected but the server response time is OK' );
+				break;
+			case 'good':
+				$result['badge']['color'] = 'green';
+				$result['label']          = __( 'Page caching is detected and the server response time is good' );
+				break;
+			default:
+				$result['badge']['color'] = 'red';
+				if ( empty( $page_cache_detail['headers'] ) && ! $page_cache_detail['advanced_cache_present'] ) {
+					$result['label'] = __( 'Page caching is not detected and the server response time is slow' );
+				} else {
+					$result['label'] = __( 'Page caching is detected but the server response time is still slow' );
+				}
+		}
+
+		$page_cache_test_summary = array();
+
+		if ( empty( $page_cache_detail['response_time'] ) ) {
+			$page_cache_test_summary[] = '<span class="dashicons dashicons-dismiss"></span> ' . __( 'Server response time could not be determined. Verify that loopback requests are working.' );
+		} else {
+
+			$threshold = $this->get_good_response_time_threshold();
+			if ( $page_cache_detail['response_time'] < $threshold ) {
+				$page_cache_test_summary[] = '<span class="dashicons dashicons-yes-alt"></span> ' . sprintf(
+					/* translators: %d is the response time in milliseconds */
```

Comment:
The translator comment here does not account for the string having more than one placeholder.